src/components/results/totalResults.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+class TotalResults extends React.Component {  
+  render() {
+    return (
+    <h2 className="doctors-total-result">Total Results: <span>{this.props.totalResults}</span></h2>
```

Comment:
[comment] For now, this will work. However, keep in mind that this value may not be present on page load, so you may need to account for a missing prop value. Look into React's [defaultProps](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) for one possible solution.